rust/osm2lanes/src/transform/tags_to_lanes/bus.rs

```diff
@@ -19,6 +19,21 @@ impl LaneBuilder {
     }
 }
 
+const BUSWAY: TagKey = TagKey::from("busway");
+
+#[derive(Debug)]
+pub struct BuswayScheme {
+    pub forward_direction: Infer<Option<Direction>>,
```

Comment:
Without looking at the implementation, this is not super intuitive...

So the forward direction is either forward or backwards,
we can infer this to some level of confidence, including inferring that it is neither forward nor backwards?

I will now read the implementation...

Reply:
I think this might just be an `Infer<bool>`

Reply:
See my [other comment](https://github.com/a-b-street/osm2lanes/pull/111#discussion_r846609663) and new name `forward_side_direction`.

If the busway scheme can't describe `forward_side_direction = Backwards` then `forward: Infer<bool>` is better. What about LHT `busway:left=opposite_lane`?

I haven't tried to understand all the assumptions and usecases for busway, so feel free to dictate or even push improvements.